core/controllers/skill_editor.py

```diff
@@ -223,8 +202,10 @@ def put(self, skill_id):
         self.render_json(self.values)
 
     @acl_decorators.can_delete_skill
-    def delete(self, skill_id):
+    def delete(self, skill_ids):
```

Comment:
Same comment here. Do not take in a lit of skill_ids, only take in one skill id

Reply:
Same as above.